deploy/helm_charts/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for ibmcloud-object-storage plugin
+name: ibmcloud-object-storage-plugin
+version: 0.1
```

Comment:
Completely agree. Since there are almost certainly more people who want to deploy the driver with Helm than just FfDL, it should make sense to maintain the helm charts in the driver repository rather than FfDL.